python/GafferSceneTest/EncapsulateTest.py

```diff
@@ -205,6 +205,21 @@ def assertHashesUnique( path ) :
 		encapsulate["in"].setInput( group1["out"] )
 		assertHashesUnique( "/group" )
 
+		# Changing the filter or globals should not affect the capsule
+		group2["in"][1].setInput( group1["out"] )
+
+		options = GafferScene.CustomOptions()
+		options["in"].setInput( group2["out"] )
+
+		encapsulate["in"].setInput( options["out"] )
+		pathFilter["paths"].setValue( IECore.StringVectorData( [ "/group/group" ] ) )
+
+		c = encapsulate["out"].object( "/group/group" )
+		pathFilter["paths"].setValue( IECore.StringVectorData( [ "/group/group", "/group/group2" ] ) )
+		self.assertEqual( c, encapsulate["out"].object( "/group/group" ) )
+		options["options"].addChild( Gaffer.NameValuePlug( "test", IECore.IntData( 10 ) ) )
+		self.assertEqual( c, encapsulate["out"].object( "/group/group" ) )
```

Comment:
Sorry, I should have realised this when we chatted before, but capsules actually _do_ [depend on the globals](https://github.com/GafferHQ/gaffer/blob/master/src/GafferScene/Capsule.cpp#L209) at present. We want to fix that somehow so they get rendered with the globals (motion blur etc) from the downstream scene, but that is a battle for another day.